### PR TITLE
test(e2e): Add LDAP Admin UI e2e test

### PR DESCRIPTION
**Pull request by moduli**
This PR adds an Admin UI e2e test that walks through setting up an LDAP auth method.

### CODE REVIEW

File: ui/admin/tests/e2e/helpers/boundary-ui.js [hunk 1]
```diff
@@ -668,9 +668,10 @@ exports.addMemberToGroup = async (page, userName) => {
 /**
  * Uses the UI to create a new role. Assumes you have selected the desired scope.
  * @param {Page} page Playwright page object
- * @param {string} roleName Name of the new role
  */
-exports.createNewRole = async (page, roleName) => {
+exports.createRole = async (page) => {
```
Comment (stasryzhov) [inline ui/admin/tests/e2e/helpers/boundary-ui.js:672]: 👍

File: ui/admin/tests/e2e/tests/auth-method-ldap.spec.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    await page.getByLabel('Description').fill('LDAP Auth Method');
+    await page.getByLabel('Address').fill(process.env.E2E_LDAP_ADDR);
+    await page.getByLabel('Bind DN').fill(process.env.E2E_LDAP_ADMIN_DN);
+    await page
+      .getByLabel('Bind Password')
+      .fill(process.env.E2E_LDAP_ADMIN_PASSWORD);
+    await page.getByRole('switch', { name: 'Discover DN' }).click();
+    await page.getByLabel('User DN').fill(process.env.E2E_LDAP_DOMAIN_DN);
+    await page.getByLabel('User Attribute').fill('uid');
+    await page.getByLabel('Group DN').fill(process.env.E2E_LDAP_DOMAIN_DN);
```
Comment (stasryzhov) [inline ui/admin/tests/e2e/tests/auth-method-ldap.spec.js:70]: Just to double check, are the User DN and Group DN supposed to be the same?

Comment (moduli) [reply ui/admin/tests/e2e/tests/auth-method-ldap.spec.js:70]: Yup. I believe it describes what part of the "LDAP tree" users and groups live. In our case, I've set up the LDAP server for users and groups to live in the same part of the tree.

**Review by DhariniJeeva — APPROVED** [review]
Thanks for getting this in! looks great

File: ui/admin/tests/e2e/tests/auth-method-ldap.spec.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    'E2E_LDAP_GROUP_NAME',
+  ]);
+
+  await checkBoundaryCli();
+});
+
+test('Set up LDAP auth method @ce @ent @docker', async ({ page }) => {
+  await page.goto('/');
+  let org;
+  let connect;
```
Comment (DhariniJeeva) [inline ui/admin/tests/e2e/tests/auth-method-ldap.spec.js:42]: not a blocker- I think these two can be `const`, I don't see them being reassigned anywhere

Comment (moduli) [reply ui/admin/tests/e2e/tests/auth-method-ldap.spec.js:42]: I believe those need to be `let`. If I change to `const`, I get this warning
```
'const' declarations must be initialized.ts (1155)
```

File: ui/admin/tests/e2e/helpers/boundary-ui.js [hunk 4]
```diff
@@ -668,9 +668,10 @@ exports.addMemberToGroup = async (page, userName) => {
...
+  const roleName = 'Credential ' + nanoid();
```
Comment (DhariniJeeva) [inline ui/admin/tests/e2e/helpers/boundary-ui.js:673]: not a blocker totally a nitpick- should this be called `Role`?

Comment (moduli) [reply ui/admin/tests/e2e/helpers/boundary-ui.js:673]: Yup! Will fix that.

**Pull request merged**